### PR TITLE
[Docs] Fix broken external links

### PR DESCRIPTION
**Pull request by marekgach**
I ran `docker run --rm dcycle/broken-link-checker:3 https://pimcore.com/docs/platform/Pimcore/  --check-extern --threads 1` to find out all the faulty external links (HTTP 404, 500).

The report is here:
[pimcore_docs_external_links.xlsx](https://github.com/pimcore/pimcore/files/13995073/pimcore_docs_external_links.xlsx)

I fix all the listed links except https://github.com/pimcore/pimcore/blob/11.x/bundles/CoreBundle/config/sitemaps.yaml where I do not know how it should be correctly. It is not only about changing the link, I think.

### CODE REVIEW

**Comment by fashxp** [comment]
thx very much!! 
could you please rebase to 11.1 so that we have these changes in the current docs too.

**Comment by marekgach** [comment]
> thx very much!! could you please rebase to 11.1 so that we have these changes in the current docs too.

@fashxp I have rebased this PR branch onto `origin/11.1`. Do you think it's okay now?

**Comment by jdreesen** [comment]
No, it contains many unrelated commits which you need to skip during the rebase.

File: doc/00_Overview/README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Most important, all elements (assets, documents or objects) can be linked and se
 
 ## ANY Channel / ANY Process
 As Pimcore is a platform that stores data independently from the channel, it can provide the managed data to any channel – simple 
-websites (B2B, B2C), commerce-systems ([integrated](https://github.com/pimcore/ecommerce-bundle/tree/1.x/doc/README.md) and/or third party), mobile apps, 
+websites (B2B, B2C), commerce-systems ([integrated](https://github.com/pimcore/ecommerce-framework-bundle/blob/1.x/README.md) and/or third party), mobile apps,
```
Comment (fashxp) [inline doc/00_Overview/README.md]:
```suggestion
websites (B2B, B2C), commerce-systems ([integrated](https://pimcore.com/docs/platform/Ecommerce_Framework/) and/or third party), mobile apps, 
```

**Comment by marekgach** [comment]
Okay, I understand now. This PR branch is based on `pimcore:11.1` with my commit atop now.

@fashxp thank you for changing the base branch. I was in the middle of asking about it. 👍

File: doc/04_Assets/03_Working_with_Thumbnails/01_Image_Thumbnails.md [hunk 2]
```diff
@@ -268,7 +268,7 @@ Pimcore supports ICC color profiles to get better results when converting CMYK i
 to RGB.
 
 Due licensing issues Pimcore doesn't include the color profiles (\*.icc files) in the download package, but
-you can download them for free here: [Adobe ICC Profiles](http://www.adobe.com/support/downloads/detail.jsp?ftpID=4075)
+you can download them for free here: [Adobe ICC Profiles](https://www.adobe.com/digitalimag/adobergb.html)
```
Comment (fashxp) [inline doc/04_Assets/03_Working_with_Thumbnails/01_Image_Thumbnails.md]:
```suggestion
you can download them for free here: [Adobe ICC Profiles](https://www.adobe.com/support/downloads/iccprofiles/iccprofiles_win.html)
```

File: doc/20_Extending_Pimcore/11_Event_API_and_Event_Manager.md [hunk 3]
```diff
@@ -26,23 +26,20 @@ All Pimcore events are defined and documented as a constant on component specifi
...
 - [Data Object Classification Store](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/DataObjectClassificationStoreEvents.php)
 - [Data Object Custom Layouts](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/DataObjectCustomLayoutEvents.php)
-- [Data Object Import](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/DataObjectImportEvents.php)
+- [Data Object Import](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/Model/DataObjectImportEvent.php)
 - [Data Object Quantity Value Unit](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/DataObjectQuantityValueEvents.php)
 - [Users / Roles](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/UserRoleEvents.php)
 - [Workflows](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/WorkflowEvents.php)
 - [Elements](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/ElementEvents.php)
 - [Mail](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/MailEvents.php)
 - [Notifications](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/NotificationEvents.php)
```
Comment (fashxp) [inline doc/20_Extending_Pimcore/11_Event_API_and_Event_Manager.md:35]:
```suggestion
- [Notifications](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/NotificationEvents.php)
- [Redirect](https://github.com/pimcore/pimcore/blob/11.x/bundles/SeoBundle/src/Event/RedirectEvents.php)
```

File: doc/20_Extending_Pimcore/11_Event_API_and_Event_Manager.md [hunk 4]
```diff
@@ -26,23 +26,20 @@ All Pimcore events are defined and documented as a constant on component specifi
...
-- [Redirect](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/RedirectEvents.php)
 - [Admin](https://github.com/pimcore/admin-ui-classic-bundle/blob/1.x/src/Event/AdminEvents.php)
 - [Frontend](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/FrontendEvents.php)
 - [Cache](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/CoreCacheEvents.php)
 - [Full-Page Cache](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/FullPageCacheEvents.php)
 - [Search](https://github.com/pimcore/pimcore/blob/11.x/bundles/SimpleBackendSearchBundle/src/Event/SearchBackendEvents.php)
 - [System](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/SystemEvents.php)
 - [Tags](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/TagEvents.php)
```
Comment (fashxp) [inline doc/20_Extending_Pimcore/11_Event_API_and_Event_Manager.md:43]:
```suggestion
- [Tags](https://github.com/pimcore/pimcore/blob/11.x/lib/Event/TagEvents.php)
- [Target Group](https://github.com/pimcore/personalization-bundle/blob/1.x/src/Event/TargetGroupEvents.php)
- [Targeting](https://github.com/pimcore/personalization-bundle/blob/1.x/src/Event/TargetingEvents.php)
```

**Review by fashxp — CHANGES_REQUESTED** [review]

**Review by fashxp — APPROVED** [review]

**Comment by fashxp** [comment]
now it looks good, thx very much

**Pull request merged**